odoo/tests/form.py

```diff
@@ -1026,6 +966,21 @@ def _cleanup_from_default(type_, value):
     return value
 
 
+def get_static_context(context_str):
+    """ Parse the given context string, and return the literal part of it. """
+    context_ast = ast.parse(context_str.strip(), mode='eval').body
+    assert isinstance(context_ast, ast.Dict)
+    result = {}
+    for key_ast, val_ast in zip(context_ast.keys, context_ast.values):
+        try:
+            key = ast.literal_eval(ast.unparse(key_ast))
+            val = ast.literal_eval(ast.unparse(val_ast))
```

Comment:
Hello

Unless I'm mistaken, this is only supported on python 3.9 but the minimal version supported by Odoo at this stage is still 3.8